chain-information.json.example

```diff
@@ -0,0 +1,34 @@
+{
+  "chainName": "<CHAIN NAME>",
+  "description": "<SHORT ONE LINE DESCRIPTION OF THE CHAIN>",
+  "chainId": ######,
+  "platformChainId": "<P CHAIN ID>",
+  "subnetId": "<SUBNET ID>",
+  "vmId": "<VM ID>",
+  "vmName": "EVM",
+  "subnetExplorerUriId": "<SHORT-CHAIN-NAME>",
+  "explorerUrl": "https://subnets.avax.network/<SHORT-CHAIN-NAME>",
+  "rpcUrl": "https://subnets.avax.network/<SHORT-URL-TAG>/testnet/rpc",
+  "wsUrl": "wss://subnets.avax.network/<SHORT-URL-TAG>/testnet/ws",
+  "isTestnet": true,
+  "networkToken": {
+    "name": "<TOKEN-NAME>",
+    "symbol": "<TOKEN-TICKER>",
+    "description": "<NETWORK NATIVE TOKEN DESCRIPTION>",
+    "decimals": 18 OR 9,
+    "logoUri": "<URL FOR TOKEN LOGO>"
+  },
+  "utilityAddresses": {
+    "multicall": "<ADDRESS OF DEPLOYED MULTICALL CONTRACT>"
+  },
+  "officialSite": "<WEBSITE>",
+  "resourceLinks": [
+    {
+      "type": "Twitter",
+      "url": "https://twitter.com/<OFFICIAL-TWITTER>"
+    }
+  ],
+  "network": "avalanche",
+  "primaryColor": "#85BC22",
+  "logoUri": "URL FOR CHAIN LOGO"
```

Comment:
@connorch up until now `primaryColor` and `logoUri` were generated fields.  (Same goes for `logoUri` for the network token)

Wondering if adding these here will be confusing.

At least the way the workflow is set up now.  This information needs to be added to the `chain-assets` repo to get synced here.

@AvaLucian @jpop32 @connorch we should sync once everyone is available about how to move forward here.

Reply:
I'm not familiar with those details so will defer to you. I just kept to the existing example of Dexalot.